Brooklyn/Resources/fr.lproj/Localizable.strings

```diff
@@ -0,0 +1,70 @@
+/* 
+  Localized.strings
+  Brooklyn
+
+  Created by Michael Charland on 2019-10-07.
+  Copyright © 2019 Pedro Carrasco. All rights reserved.
+*/
+
+"Apple's Original Version" = "Version originale d'Apple";
+"Avatar Tissue"= "Tissu Avatar";
```

Comment:
Can we make the keys a little more formatted?
For example: `animation.appleOriginalVersion`

Reply:
Great idea, I found a much simpler way to do this. Hope you like it :) In Animation use the enum value as the key and look that up in the Localizable.strings files. It makes the Animation enum super simple :)

Reply:
I pushed up the change just for the Animations will do the menus and the rest of the string when I find a good solution to deal with xib based localizations.

Reply:
> do the menus and the rest of the string when I find a good solution to deal with xib based localizations.

Regarding this, move them to this file with good key names. I'm not usual a XIB user (I tend to do things by code, except in Brooklyn 😅) but I would assume you'll have to associate localizable strings to an UI element in code.

Reply:
https://medium.com/@mario.negro.martin/easy-xib-and-storyboard-localization-b2794c69c9db This doesn't look bad either

Reply:
Great suggestion. I've implemented and agree it's much cleaner and long term maintainable :)